### ISSUE TITLE
Built-in SCSS validation is erroneously reporting nested @support blocks

### ISSUE THREAD

**Issue by brianblakely** [Microsoft/vscode#39273]
- VSCode Version: Code 1.18.1 (929bacba01ef658b873545e26034d1a8067445e9, 2017-11-16T18:23:26.125Z)
- OS Version: Darwin x64 16.7.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
code-settings-sync|Sha|2.8.6
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
vscode-eslint|dba|1.4.3
vscode-npm-script|eg2|0.3.3
final-newline|sam|0.3.2
shader|sle|1.1.2

---

Steps to Reproduce:

**Create a SCSS file with this content:**

```scss
#my-element {
  color: red;

  @supports (color: blue) {
    color: blue;
  }
}
```

**Result**

The validator reports a problem with this code: `[scss] } expected`.

Reproduces without extensions: Yes

**Comment by pinksoir** [comment]
Same issue ocurring here:

![vsc_error](https://user-images.githubusercontent.com/29377406/33335968-8532d648-d466-11e7-8730-ee42a841f954.jpg)

Occurs with or without extensions..

**Comment by aeschli** [comment]
duplicate of #35887, fixed in 1.19
